.travis.yml

```diff
@@ -90,8 +90,9 @@ matrix:
           before_install:
               - nvm install 8
               - nvm use 8
-              - docker pull trzeci/emscripten:sdk-tag-1.35.4-64bit
+              - docker pull trzeci/emscripten:sdk-tag-1.37.21-64bit
```

Comment:
So does it needs an emscripten update then?

Reply:
Yes - only updating both emscripten and boost currently works - updating either one alone still results in errors that we don't have a solution for so far. However, this only updates the emscripten version on travis to the same version used on circleci and does not update any further than that.

Reply:
Ah, in that case I'm not sure we should merge without running some benchmarks. It is kind of weird, because on circle 1.37.21 compiles with both versions of boost, while on travis 1.37.21 needs the boost upgrade.

Well, we could use the circle output to benchmark against he nightlies, since both versions were created.

Reply:
I think on circle everything is compiled in docker containers, whereas on travis parts are built in a docker container and parts are not - my conjecture would be that that's causing the differences. I'd have to recheck, though.